.gitignore

```diff
@@ -37,4 +37,7 @@ tests/proguard-project.txt
 .gradle
 .idea
 *.iml
-build
\ No newline at end of file
+build
+
+# android sdk captures folder
+captures
```

Comment:
Where is this directory located? I do not see it in the app directories.

Reply:
Its not there, but if you create captures it will be auto generated. However in order to not evidently upload captures we need to place it here.